src/main/java/net/minecraftforge/common/brewing/BrewingRecipeRegistry.java

```diff
@@ -168,4 +167,80 @@ public static boolean isValidInput(ItemStack stack)
     {
         return Collections.unmodifiableList(recipes);
     }
+
+    /**
+     * Returns an unmodifiable list of all fuels in the registry.
+     */
+    public static List<FuelEntry> getFuels()
+    {
+        return Collections.unmodifiableList(fuels);
+    }
+
+    /**
+     * Adds a new fuel for brewing stands.
+     */
+    public static void addFuel(ItemStack item, int fuelValue)
+    {
+        fuels.add(new FuelEntry(item, fuelValue));
+    }
+
+    /**
+     * Check whether an item is a fuel of brewing stands.
+     */
+    public static boolean isFuel(@Nullable ItemStack item)
+    {
+        if (item == null)
+        {
+            return false;
+        }
+        for (FuelEntry e : fuels)
+        {
+            if (e.getItem().isItemEqual(item))
+            {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Gets the fuel value of an item.
+     */
+    public static int getFuelValue(@Nullable ItemStack fuel)
+    {
+        if (fuel == null)
+        {
+            return 0;
+        }
+        for (FuelEntry e : fuels)
+        {
+            if (e.getItem().isItemEqual(fuel))
+            {
+                return e.getFuelValue();
```

Comment:
Perhaps this should be implemented such that the highest fuel value of all registered fuel values is the one that is used, similarly to how normal Furnace fuels work. ([`GameRegistry#getFuelValue`](https://github.com/MinecraftForge/MinecraftForge/blob/1.9/src/main/java/net/minecraftforge/fml/common/registry/GameRegistry.java#L269-L277))

This way, if multiple mods add say, blaze rods as a fuel but one for 60 ticks and one for 80 ticks, the 80 tick one takes precedence over the 60 tick one instead of whichever one is registered first being used.